circuit/asset_delta.go

```diff
@@ -63,6 +75,17 @@ func UpdateAccounts(
 	return AccountsInfoAfter
 }
 
+func GetGasDeltas(gasFeeAssetId, gasFeeAssetAmount Variable) (
+	gasDeltas [NbGasAssetsPerTx]GasDeltaConstraints) {
+	gasDeltas[0].AssetId = gasFeeAssetId
+	gasDeltas[0].BalanceDelta = gasFeeAssetAmount
+
+	for i := 1; i < NbGasAssetsPerTx; i++ {
+		gasDeltas[i] = EmptyGasDeltaConstraints()
```

Comment:
In this case,  `gasDeltas[1]` will be always empty.  I was thinking there is no `AssetId` field, and `gasDeltas[0]` represent `BNB`, `gasDeltas[0]`  represent `BUSD`